tests/gordo_components/workflow/test_server_to_sql/test_server_to_sql.py

```diff
@@ -1 +1,10 @@
-# TODO: Add relevant tests
+import tests.utils as tu
+from gordo_components.workflow.server_to_sql import server_to_sql as sts
+
+
+def test_get_machines_from_server(ml_server):
+    machines = sts.get_machines_from_server(
```

Comment:
This seems to unnecessarily add a bit of strain looking over there tests. I don't know what the other's opinions are on this namespacing stuff, but I am not really a fan of `np` or `pd` either.

Reply:
Looking at 'sts' is more straining than a free function? To me, much like the well accepted practice of shortnaming pandas and numpy, looking at a prefix immediately shows me this function is from another module. Thus I don't have to think "is this a function defined in the current file?"